app/helpers/application_helper.rb

```diff
@@ -41,6 +41,10 @@ def icon(name)
     image_tag "icons/#{name}.png"
   end
 
+  def ldate(date, options = {})
```

Comment:
I'd rather keep the more specific `event_start_time` method, until we have more users for a generic `ldate` method with two arguments and a somewhat cryptic name.